### PR TITLE
CBMC v6 fixes and release 3.9

### PR DESCRIPTION
**Pull request by tautschnig**
See individual commit messages for details.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by rod-chapman** [comment]
Could you drop a release 3.9 into Homebrew as soon as you have it please?

**Comment by rod-chapman** [comment]
s2n-tls's CI runners install this via "pip3 install cbmc-viewer==$CBMC_VIEWER_VERSION" where CBMC_VIEWER_VERSION is pulled from https://api.github.com/repos/model-checking/cbmc-viewer/releases/latest

Will that automatically get updated too?

File: src/cbmc_viewer/tracet.py [hunk 1]
```diff
@@ -329,7 +329,7 @@ def parse_xml_traces(xmlfile, root=None):
     if xml.find('result') is not None:
         for line in xml.iter('result'):
             name, status = line.get('property'), line.get('status')
-            if status == 'SUCCESS':
+            if status == 'SUCCESS' or status == 'UNKNOWN':
```
Comment (feliperodri) [inline src/cbmc_viewer/tracet.py:332]: Would that make any existing warning silent?

**Review by feliperodri — APPROVED** [review]

**Comment by feliperodri** [comment]
It seems now we are failing to produce `result.xml` for some cases in coreHTTP.

File: src/cbmc_viewer/tracet.py (hunk 1 above)
Comment (tautschnig) [reply src/cbmc_viewer/tracet.py:332]: No, there never was a warning about this, we now just have unsuccessful attempts to extract a trace when the status is not `SUCCESS`.

**Comment by tautschnig** [comment]
> It seems now we are failing to produce `result.xml` for some cases in coreHTTP.

I will investigate what is going on there.

**Comment by tautschnig** [comment]
> > It seems now we are failing to produce `result.xml` for some cases in coreHTTP.
> 
> I will investigate what is going on there.

We will first need to fix coreHTTP to work with CBMC v6 (updating the use of `--object-bits`). Merging as the failure is unrelated.

**Comment by tautschnig** [comment]
> s2n-tls's CI runners install this via "pip3 install cbmc-viewer==$CBMC_VIEWER_VERSION" where CBMC_VIEWER_VERSION is pulled from https://api.github.com/repos/model-checking/cbmc-viewer/releases/latest
> 
> Will that automatically get updated too?

@rod-chapman This works now.

**Comment by tautschnig** [comment]
> Could you drop a release 3.9 into Homebrew as soon as you have it please?

Homebrew should be available as soon as https://github.com/aws/homebrew-tap/pull/557 has been merged.

**Pull request merged**